### PR TITLE
Live singles in ci test

### PR DESCRIPTION
**Pull request by GarethCabournDavies**
Part of testing for https://github.com/gwastro/pycbc/pull/3984 is that we wanted to get the single-detector trigger into the live test

In this PR, we:
- Set the H1 and V1 strain generator to end 32s early
- Add another injection to inject an EM-bright signal into this (L1-only) time
- Make a single fits file (currently just manually filling the appropriate datasets to get something sensible)
- Add the singles to the test `run.sh` script
- Update the `check_results.py` script to know that there may be single-detector events (and call them 'found' rather than 'coinc')

### CODE REVIEW

**Comment by GarethCabournDavies** [comment]
I don't seem to be able to request @maxtrevor to do a review?

File: examples/live/make_singles_fits_file.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# This script will make a valid singles-fits file for use in the
+# pycbc_live CI tests. It doesn't have much physical meaning,
+# but will give broadly representative numbers for singles
```
Comment (titodalcanton) [inline examples/live/make_singles_fits_file.py:3]:
```suggestion
"""
This script will make a valid singles-fits file for use in the
pycbc_live CI tests. It doesn't have much physical meaning,
but will give broadly representative numbers for singles.
"""
```

**Comment by titodalcanton** [comment]
I suggest changing `--frame-read-timeout 100` to something like 10 to avoid having the example stall for 100 s waiting for the final bits of the frame files.

**Comment by maxtrevor** [comment]
> I don't seem to be able to request @maxtrevor to do a review?

Hi Gareth, I was traveling before but I can take a look at this now

**Comment by GarethCabournDavies** [comment]
> > I don't seem to be able to request @maxtrevor to do a review?
> 
> Hi Gareth, I was traveling before but I can take a look at this now

Thanks!

File: examples/live/make_singles_fits_file.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+"""
```
Comment (maxtrevor) [inline examples/live/make_singles_fits_file.py:1]: Just making sure I understand, this file is used to calculate the significance of single detector triggers? It represents the results of fitting an exponential to the background distribution of noise triggers?

**Review by maxtrevor — COMMENTED** [review]
I think the changes to check_results.py , generate_injections.py and run.sh all look reasonable. I just want to make sure I understand what the make_single_fits_file.py is for.

File: examples/live/make_singles_fits_file.py (hunk 2 above)
Comment (GarethCabournDavies) [reply examples/live/make_singles_fits_file.py:1]: I think @titodalcanton has the correct rights

**Comment by GarethCabournDavies** [comment]
@titodalcanton - I think we just need someone to click 'approve' on this now

**Review by titodalcanton — APPROVED** [review]
Thanks @GarethCabournDavies and sorry for the delay. This looks ok.

**Pull request merged**